requirements-dev.txt

```diff
@@ -4,3 +4,5 @@ python-jose
 scriptworker
 signtool>=3.1.6
 taskcluster
+requests_hawk==1.0.0
```

Comment:
I wonder if there's a need for an aiohttp_hawk :)

Reply:
Looked into this briefly on Friday.

The aiohttp methods expect an [aiohttp.BasicAuth](https://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.BasicAuth) for the auth param, but we could [use mohawk to construct the header directly](https://mohawk.readthedocs.io/en/latest/usage.html#sending-a-request) and add the Authorization to the headers dict.

This would let us drop requests and requests_hawk.

Reply:
Ok. I don't think this blocks. If you want to tackle that, great; if not, and if using mohawk+aiohttp is preferred, we can leave as a followup.